api/main.py

```diff
@@ -660,7 +698,10 @@ def validate_measurement_data(data):
     return True
 
 @app.post('/v1/hog/add')
-async def hog_add(measurements: List[HogMeasurement]):
+async def hog_add(
+    measurements: List[HogMeasurement],
+    user: User = Depends(authenticate), # pylint: disable=unused-argument
+    ):
```

Comment:
The way this is currently implemented if there is no header present the default user will be user. We need to make sure that data is not shared per accident. The same with carbonDB. Let's say I start the hog and don't add a key all my process data will be uploaded with the default user. This will happen with everyone, so we need to make sure that the data can not be retrieved for the default user. Or the default user can't access the hog and carbondb add api endpoints.

Reply:
But why? This was the behaviour before ...? If you do not use a token your data will get pooled. I think this is perfectly fine ...?

Happy for your feedback. In any case it is nothing to be done in code. This is done via the DB data

Reply:
No, before every instance generated a unique UUID that was used to identify a user.